server/mailet/mock-smtp-server/pom.xml

```diff
@@ -119,4 +119,84 @@
             <version>3.1.7</version>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>1.5.0</version>
+                <configuration>
+                    <from>
+                        <image>openjdk:8u222-jre</image>
+                    </from>
+                    <to>
+                        <image>linagora/mock-smtp-server</image>
+                        <tags>
+                            <tag>0.1</tag>
+                            <tag>latest</tag>
+                        </tags>
+                    </to>
+                    <container>
+                        <mainClass>org.apache.james.mock.smtp.server.MockSMTPServerMain</mainClass>
+                        <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
```

Comment:
for what?

Reply:
Because if you don't, your image gets its creation time set up to Unix Epoch, so something like... 49 years old? It's in the FAQ: https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#why-is-my-image-created-48-years-ago

Reply:
Didn't think about it that way, but makes sense yes thanks !